src/main/java/io/opentracing/contrib/global/GlobalTracer.java

```diff
@@ -0,0 +1,115 @@
+package io.opentracing.contrib.global;
+
+import io.opentracing.NoopTracer;
+import io.opentracing.NoopTracerFactory;
+import io.opentracing.Span;
+import io.opentracing.Tracer;
+import io.opentracing.contrib.global.delegation.DelegateTracer;
+
+import java.util.concurrent.Callable;
+import java.util.concurrent.atomic.AtomicReference;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+/**
+ * The <code>GlobalTracer</code> class is not a {@link Tracer} implementation of itself, but instead a utility-class
```

Comment:
I like the term _ActiveTracer_ but share the concern with @yurishkuro that is comes accross weak to have to explain why something called ..Tracer is in fact not a `Tracer`.
I could also just remove that from the Doc but wanted to be upfront about it.

The `GlobalTracer` name was directly influenced from the _Global and No-op Tracers_ section from the _Concepts and Terminology_ OT documentation.
For consistency I'd prefer the term GlobalTracer to be represented in code and obtaining its instance to be as intuitive as possible.

Any ideas?